sql/init.sql

```diff
@@ -1,6 +1,6 @@
 ADD SHARDING RULE r1 COLUMNS w_id;
 ADD SHARDING RULE r2 COLUMNS id;
 ADD SHARDING RULE r3 COLUMNS bid;
-ADD KEY RANGE krid3 FROM 21 TO 30 ROUTE TO sh2;
+ADD KEY RANGE krid3 FROM 21 TO 9223372036854775807 ROUTE TO sh2;
```

Comment:
Could you please add a comment here about 9223372036854775807?
Smth like `-- BIGSERIAL type is from 0 to X`